src/pages/home/report/ReactionList/BaseReactionList.tsx

```diff
@@ -1,64 +1,64 @@
 /* eslint-disable rulesdir/onyx-props-must-have-default */
 import Str from 'expensify-common/lib/str';
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FlatList} from 'react-native';
+import type {FlatListProps} from 'react-native';
 import OptionRow from '@components/OptionRow';
-import participantPropTypes from '@components/participantPropTypes';
-import withWindowDimensions from '@components/withWindowDimensions';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import Navigation from '@libs/Navigation/Navigation';
 import * as UserUtils from '@libs/UserUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
+import type {PersonalDetails} from '@src/types/onyx';
 import HeaderReactionList from './HeaderReactionList';
-import reactionPropTypes from './reactionPropTypes';
+import type ReactionListProps from './types';
 
-const propTypes = {
+type BaseReactionListProps = ReactionListProps & {
     /**
      *  Array of personal detail objects
      */
-    users: PropTypes.arrayOf(participantPropTypes).isRequired,
+    users: PersonalDetails[];
 
     /**
      * Returns true if the current account has reacted to the report action (with the given skin tone).
      */
-    hasUserReacted: PropTypes.bool,
+    hasUserReacted: boolean;
```

Comment:
This isn't a required prop:
```suggestion
    hasUserReacted?: boolean;
```

Reply:
@blazejkustra Quick question:
Are you sure about this? Because this property is coming from `EmojiUtils.getEmojiReactionDetails` which is not optional there. Plus later it's getting passed down to `getEmojiReactionBubbleStyle` and `getEmojiReactionCounterTextStyle` methods of the `useStyleUtils` hook, which again need this as a mandatory argument. If you're sure this shouldn't be reqired, please let me know and I will proceed to pass a default value down those methods instead.

Reply:
Let's use a default value instead and make this optional. When you add a default value `getEmojiReactionDetails ` won't complain

Reply:
Done. I also remove passing down the `onClose` prop to the `HeaderReactionList` component, since it wasn't getting used there at all.